### ISSUE TITLE
[Webextension Regression] - Calendar invites not handled by calendar

### ISSUE THREAD

**Issue by dugite-code** [TB-throwback/LookOut-fix-version#104]
This was a regression introduced with the migration to webextension. ICS files are extracted but not accessible directly via the standard Calendar accept/decline interface.

**Comment by vsessink** [comment]
The bug says "closed" but I'm still seeing ".ics" files instead of just the "Lightning" interface showing an invitation. On Linux, luckily, Thunderbird proposes to open this ics-file with Thunderbird, importing the invitation. However MacOS wants "CalendarFileHandler" to open it - delegating calendar management to Apple Calendar.
This is with Thunderbird 115.6.1 with Lookout 6.1.

**Comment by dugite-code** [comment]
Yes the previous issue was a bug in the implementation and was closed as it pertained to an old version of the extension. The new issue is a regression and a new method is required to achieve a similar result as was done in the past.

**Comment by dugite-code** [comment]
Implemented in 7.4
